tasks/subscription-manager.yml

```
@@ -47,6 +47,38 @@
            and __rhc_subman_identity.rc == 1
            and rhc_release != __rhc_state_absent )
       else omit }}"
+    server_proxy_hostname: "{% if rhc_proxy is none -%}
+      {{ omit }}
+      {%- elif rhc_proxy == {} -%}
+      {%- elif 'hostname' not in rhc_proxy -%}
```

```suggestion
      {%-   elif 'hostname' not in rhc_proxy -%}
```
indent for readability